velox/exec/TableWriter.cpp

```diff
@@ -91,13 +96,55 @@ void TableWriter::addInput(RowVectorPtr input) {
   }
   dataSink_->appendData(mappedInput);
   numWrittenRows_ += input->size();
+  if (aggregation_ != nullptr) {
+    aggregation_->addInput(input);
+  }
 }
 
+/// Both the statistics and the row_count + fragments are transferred over the
```

Comment:
Nice comment. Could you move to TableWriterTraits. Also comment in cc starting with '//'.